corehq/apps/domain/forms.py

```diff
@@ -1223,6 +1223,13 @@ def current_account(self):
     def current_subscription(self):
         return Subscription.get_subscribed_plan_by_domain(self.domain)[1]
 
+    @property
+    @memoized
+    def current_contact_emails(self):
+        return BillingContactInfo.objects.get_or_create(
+            account=self.current_subscription.account
+        )[0].emails
```

Comment:
I personally dislike the pattern of sneakily hiding things that write to the database in what look like property accessors as it can easily trip people up. Might suggest naming this to `get_or_create_current_contact_emails` and making it a function instead of a property (I realize that's more verbose and also not a perfect name since you're making a `BillingContactInfo` but couldn't quickly come up with a better one). 

Not blocking merge - @benrudolph feel free to merge if you're cool with this.